### PR TITLE
refactor: breakpoints-page-sections/faq

### LINKED ISSUE #313: Fix container query in `faq`
**Provide the exact quote of the error or issue:**

We should modify the `_container` query to use breakpoints provided by `@yamada-ui/react`.

`contents\page-sections\faq\with-image\index.tsx`

This issue will be automatically unassigned after two weeks have passed since it was assigned. Once unassigned, it may be picked up by someone else for work.

### PR DESCRIPTION
**Pull request by bearone236**
Closes #313

### CODE REVIEW

File: contents/page-sections/faq/with-image/index.tsx [hunk 1]
```diff
@@ -11,44 +11,15 @@ import FAQ from "./faq.svg"
...
     <Flex
-      _container={[
-        {
-          maxW: "900px",
-          css: {
-            flexDir: "column",
-          },
-        },
-      ]}
+      flexDir={{ base: "row", md: "column" }}
```
Comment (illionillion) [inline contents/page-sections/faq/with-image/index.tsx:14]: Isn't it equivalent to this?

```suggestion
      flexDir={{ lg: "column" }}
```

File: contents/page-sections/faq/with-image/index.tsx [hunk 2]
```diff
@@ -11,44 +11,15 @@ import FAQ from "./faq.svg"
...
-        _container={[
-          {
-            maxW: "900px",
-            css: {
-              maxW: "100%",
-            },
-          },
-        ]}
-      >
+      <Image src={FAQ.src} alt="faq" w={{ base: "3xl", md: "full" }} />
```
Comment (illionillion) [inline contents/page-sections/faq/with-image/index.tsx:21]:
```suggestion
      <Image src={FAQ.src} alt="faq" w={{ base: "lg", sm: "full" }} />
```

File: contents/page-sections/faq/with-image/index.tsx [hunk 3]
```diff
@@ -11,44 +11,15 @@ import FAQ from "./faq.svg"
...
+      <VStack maxW={{ base: "full", md: "3xl" }}>
```
Comment (illionillion) [inline contents/page-sections/faq/with-image/index.tsx:22]:
```suggestion
      <VStack maxW={{ base: "xl", sm: "full" }}>
```

**Review by illionillion — COMMENTED** [review]
@bearone236 

I think this way fits nicely on the screen.

https://github.com/user-attachments/assets/c51f2278-0102-43f4-a6a5-5fad862c5007

**Review by illionillion — APPROVED** [review]
LGTM

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**